app/src/app/components/stix/stix-list/stix-list.component.html

```diff
@@ -90,15 +90,23 @@
                     </td>
                 </ng-container>
 
-                <!-- open in new icon column (for relationships) -->
-                <ng-container matColumnDef="open-link" stickyEnd>
-                    <th mat-header-cell *matHeaderCellDef>
-                        <!-- nothing in header -->
-                    </th>
+                <!-- open new dialog icon column (for relationships) -->
+                <ng-container matColumnDef="open-dialog" stickyEnd>
+                    <th mat-header-cell *matHeaderCellDef></th> <!-- nothing in header -->
                     <td mat-cell *matCellDef="let element" class="icon-cell">
-                        <mat-icon aria-hidden="false" aria-label="open">open_in_new</mat-icon>
+                        <mat-icon aria-hidden="false" aria-label="open-dialog"
+                          matTooltip="Info about the action">expand_content</mat-icon>
```

Comment:
We may want to keep this as a `open_in_new` icon to keep it consistent with how we've been using it as an `open in dialog` icon.

Also recommend updating the tooltip from

> Info about the action

to

>  View object in dialog

<img width="866" alt="image" src="https://user-images.githubusercontent.com/48832936/225699000-29644e88-5083-4da4-8287-eea61094f458.png">